### ISSUE TITLE
Error: DEM: Provide a value for non-optional argument! Error: Elevation

### ISSUE THREAD

**Issue by angelini75** [stevenpawley/Rsagacmd#24]
Hi there. Nice package, thanks.

I got the following error
```
> library(Rsagacmd) 
> library(terra) > 
> # initiate a saga object 
> saga = saga_gis(saga_bin = "C:/Program Files/SAGA/saga_cmd.exe", 
>                            raster_backend = "terra", 
>                            vector_backend = "sf") 
>  
> # load the example data 
> srtm <- read_srtm() 
> plot(srtm) # ok!
>  
> # access the libraries and tools 
> saga$ta_morphometry$topographic_position_index_tpi(dem = srtm) 
Error in `saga_execute()`: ! Error: DEM: Provide a value for non-optional argument! 
Error: Elevation Run `rlang::last_trace()` to see where the error occurred. 
Called from: signal_abort(cnd, .file) 
Browse[1]
```
SAGA-GIS 9.0.0

**Comment by jannes-m** [comment]
I see the same with SAGA-GIS 8.5.0

**Comment by stevenpawley** [comment]
Thanks for reporting this, will follow up, maybe something has changed with the SAGA command line tool.

**Comment by stevenpawley** [comment]
Looks like saga_cmd only accepts '=' rather than the traditional ':' for the command arguments. Hopefully, this is fixed now by 1cc5405
